runtime/lua/vim/diagnostic.lua

```diff
@@ -429,32 +429,30 @@ local function get_diagnostics(bufnr, opts, clamp)
     end
   end
 
+  local add_iterator = function(buf_nr, diag_tbl)
```

Comment:
In addition to justinmk's comment, use `bufnr` instead of `buf_nr`. Minor nit, but we use `bufnr` everywhere so it's more consistent.

```suggestion
  local add_iterator = function(bufnr, diag_tbl)
```

Reply:
why i don't use bufnr because there already define variable with this name. so sumneko lua will give a hint `Redefined`. a noise .. but I changed now ..

Reply:
rename to `buf` .. `bufnr` will make ci  check failed